CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+cmake_minimum_required(VERSION 3.3.1)
+
+project(pms CXX)
```

Comment:
seeing that this project only does C++, add `LANGUAGES CXX` inside parentheses here to minimise configure churn